src/AppBundle/Resources/views/Admin/Client/Report/partials/_prof-deputy-costs.html.twig

```diff
@@ -0,0 +1,129 @@
+{% set translationDomain = "admin-checklist" %}
+{% trans_default_domain translationDomain %}
+{% set page = 'checklistPage' %}
+{% set hideEditLink = true %}
+
+<h3 class="heading-medium underline" id="profDeputyCosts">{{ (page ~ '.heading.profDeputyCosts') | trans }}</h3>
+
+<div class="form-section push--bottom">
+    <div class="form-group-outer">
+        {{ form_checkbox_group(form.paymentsMatchCostCertificate, (page ~ '.form.paymentsMatchCostCertificate'), {
+            'fieldSetClass' : 'inline'
+        }) }}
+    </div>
+
+    <details>
+        <summary>
+            <span class="summary">{{ (page ~ '.revealTitle.profDeputyCosts') | trans }}</span>
+        </summary>
+        <div class="panel panel-border-narrow">
+            <div>
+
+                <table class="check-your-answers columns-2 push--bottom">
+                    {{ macros.summaryTableHeaderQA(hideEditLink) }}
+                    <tbody>
+
+                {% include 'AppBundle:Report/ProfDeputyCosts:_payments_received_answers.html.twig' with {
+                    'report': report,
+                    'transDomain': "report-prof-deputy-costs",
+                    'hideEditLink' : hideEditLink
+                } %}
+                    </tbody>
+                </table>
+
+            </div>
+        </div>
+    </details>
+
+</div>
+
+<div class="form-section push--bottom">
+    <div class="form-group-outer">
+        {{ form_checkbox_group(form.profCostsReasonableAndProportionate, (page ~ '.form.profCostsReasonableAndProportionate'), {
+            'fieldSetClass' : 'inline'
+        }) }}
+    </div>
+
+    <details>
+        <summary>
+            <span class="summary">{{ (page ~ '.revealTitle.profDeputyCosts') | trans }}</span>
+        </summary>
+        <div class="panel panel-border-narrow">
+            <div>
+                <table class="check-your-answers columns-2 push--bottom">
+                    {{ macros.summaryTableHeaderQA(hideEditLink) }}
+                    <tbody>                
+                    {% include 'AppBundle:Report/ProfDeputyCosts:_how_charged_answers.html.twig' with {
+                        'report': report,
+                        'transDomain': 'report-prof-deputy-costs',
+                        'hideEditLink' : true
+                    } %}
+                    {% include 'AppBundle:Report/ProfDeputyCosts:_all_cost_total_answers.html.twig' with {
+                        'report': report,
+                        'submittedOtherCosts': report.generateActualSubmittedOtherCosts,
```

Comment:
This calculation is being performed several times, can we `set actualOtherCosts = ...`?

Reply:
I'll look into this